engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/CallCacheDiffActor.scala

```diff
@@ -4,15 +4,14 @@ import akka.actor.{ActorRef, LoggingFSM, Props}
 import cats.data.NonEmptyList
 import cats.instances.list._
 import cats.syntax.foldable._
-import cromwell.core.WorkflowId
+import cromwell.core.Dispatcher.EngineDispatcher
```

Comment:
ToL: Huh... was this unused `WorkflowId` not flagged by our build flags?

Reply:
Apparently not. I don't get it either. 😦